sqlmesh/core/test/definition.py

```diff
@@ -63,15 +63,17 @@ def setUp(self) -> None:
             name: self._get_rows(table) for name, table in self.body.get("inputs", {}).items()
         }
 
-        for table, rows in inputs.items():
+        for table_name, rows in inputs.items():
             df = pd.DataFrame.from_records(rows)  # noqa
             columns_to_types: dict[str, exp.DataType] = {}
             for i, v in rows[0].items():
                 # convert ruamel into python
                 v = v.real if hasattr(v, "real") else v
                 columns_to_types[i] = parse_one(type(v).__name__, into=exp.DataType)
-            self.engine_adapter.create_schema(table)
-            self.engine_adapter.create_view(_test_fixture_name(table), df, columns_to_types)
+            table = exp.to_table(table_name)
+            if table.db:
```

Comment:
This is actually a bugfix. Before if you passed in a name that was just a table name then we would actually create that name as a schema but then correctly create the view where you would expect.

Reply:
should we fix this in crceate_schema?

Reply:
Well if `create_schema` is just given a single name then it assumes this a schema name which makes sense. So I think the bug is the caller passing in a table into `create_schema` instead of a schema.

Reply:
I agree with Ryan. The logic of automatically retrieving a schema if a table name was passed always felt kind of wrong to me. Explicit better than implicit